subprojects/docs/src/docs/userguide/tutorial/part5_gradle_caching.adoc

```diff
@@ -0,0 +1,206 @@
+// Copyright 2017 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[part5_gradle_caching]]
+= Part 5: Gradle Cache
```

Comment:
I miss an explanation of how the build cache is different from incremental builds. I think this part should cover both and start with the incremental builds as the more basic optimization.

Reply:
Fixed in latest commit.